x-pack/plugins/security/server/plugin.test.ts

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
__note:__ just a couple of basic tests, mostly a ground work for future tests for this file that we'll surely have.